tests/components/rfxtrx/__init__.py

```diff
@@ -4,6 +4,11 @@
 
 async def _signal_event(hass, packet_id):
     event = rfxtrx.get_rfx_object(packet_id)
-    hass.helpers.dispatcher.async_dispatcher_send(rfxtrx.SIGNAL_EVENT, event)
+
+    await hass.async_add_executor_job(
+        hass.data[rfxtrx.DATA_RFXOBJECT].event_callback, event,
```

Comment:
Can we get the callback from the `call_args` of the patched mock connection? Then we don't need to dig into hass.data which is an integration details.

Reply:
Would need to pass that along everywhere the function is called. I'd rather not in this pull if okey. It's already grown with unrelated things. But in principle I agree. Can run a separate pass with that somewhat later.

Reply:
Ok, good.